vscode_projects/__init__.py

```diff
@@ -61,7 +61,7 @@ def handleQuery(self, query):
                 uri = urllib.parse.unquote(uri)
                 results.append(
                     Item(
-                        id="kill",
+                        id=md_id,
```

Comment:
this should be the identifier of a project item right? identifiers have to be unique.